sdk/storage/azqueue/README.md

```diff
@@ -0,0 +1,243 @@
+
+// 2. enqueue message
+_, err = queueClient.EnqueueMessage(context.TODO(), message, nil)
+// TODO: handle error
+
+// 3. dequeue message
+_, err = queueClient.DequeueMessage(context.TODO(), nil)
+// TODO: handle error
+
+// 4. delete queue
+_, err =queueClient.Delete(context.TODO(), nil)
+// TODO: handle error
+```
+
+### Enumerating queues
+
+```go
+const (
+	account = "https://MYSTORAGEACCOUNT.queue.core.windows.net/"
+)
+
+// authenticate with Azure Active Directory
+cred, err := azidentity.NewDefaultAzureCredential(nil)
+// TODO: handle error
+
+// create a client for the specified storage account
+client, err := azqueue.NewServiceClient(account, cred, nil)
+// TODO: handle error
+
+// queue listings are returned across multiple pages
+pager := client.NewListQueuesPager(nil)
+
+// continue fetching pages until no more remain
+for pager.More() {
+   resp, err := pager.NextPage(context.Background())
+   _require.Nil(err)
+   // print queue name
+   for _, queue := range resp.Queues {
+		fmt.Println(*queue.Name)
+	}
+}
+```
+
+## Troubleshooting
+
+All queue service operations will return an
+[*azcore.ResponseError][azcore_response_error] on failure with a
+populated `ErrorCode` field. Many of these errors are recoverable.
+The [queueerror][queue_error] package provides the possible Storage error codes
+along with various helper facilities for error handling.
+
+```go
+const (
+	connectionString = "<connection_string>"
+	queueName        = "samplequeue"
+)
+
+// create a client with the provided connection string
+client, err := azqueue.NewServiceClientFromConnectionString(connectionString, nil)
+// TODO: handle error
+
+// try to delete the queue, avoiding any potential race conditions with an in-progress or completed deletion
+_, err = client.DeleteQueue(context.TODO(), queueName, nil)
+
+if queueerror.HasCode(err, queueerror.QueueBeingDeleted, queueerror.QueueNotFound) {
+	// ignore any errors if the queue is being deleted or already has been deleted
+} else if err != nil {
+	// TODO: some other error
+}
+```
+
+## Next steps
+
+Get started with our [Queue samples][samples].  They contain complete examples of the above snippets and more.
+
+## Contributing
+
+See the [Storage CONTRIBUTING.md][storage_contrib] for details on building,
+testing, and contributing to this library.
+
+This project welcomes contributions and suggestions.  Most contributions require
+you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution. For
+details, visit [cla.microsoft.com][cla].
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc].
+For more information see the [Code of Conduct FAQ][coc_faq]
+or contact [opencode@microsoft.com][coc_contact] with any
+additional questions or comments.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-go%2Fsdk%2Fstorage%2Fazblob%2FREADME.png)
```

Comment:
References `azblob`

Reply:
its just a PNG, I think ill just remove this impressions tag altogether